docs/pages/deployment/monitoring.rst

```diff
@@ -54,6 +61,13 @@ Returns the status of the various services in ``yaml`` format:
 
 If you supply ``application/json`` for the ``Accept`` HTTP header it will return the diagnostics in JSON format.
 
+Explanation of ambiguous/complex entries in the diagnostics:
```

Comment:
Should value of -1 be explained?

Reply:
Not really I think, there will be an error in the log